Examples/Scripts/Python/ckf_tracks.py

```diff
@@ -91,7 +95,18 @@ def runCKFTracks(
         trackingGeometry,
         field,
         TruthSeedRanges(pt=(500.0 * u.MeV, None), nHits=(9, None)),
-        ParticleSmearingSigmas(pRel=0.01),  # only used by SeedingAlgorithm.TruthSmeared
+        ParticleSmearingSigmas(  # only used by SeedingAlgorithm.TruthSmeared
+            d0=0,
+            d0PtA=0,
+            d0PtB=0,
+            z0=0,
+            z0PtA=0,
+            z0PtB=0,
+            t0=0,
+            phi=0,
+            theta=0,
+            ptRel=0,
+        ),
```

Comment:
Can you explain why these are now all 0? 
Better to set 0 as default instead?

Reply:
I think at the end it is a choice and depends on what we want to do. For track fitting this is fine because we will constrain to find the measurements. For track finding this can lead to problems so starting with the true parameters makes more sense to me. I guess the questions is whether to use truth smeared seeding with the finding in the first place.

Reply:
added a comment